insights/specs/default.py

```diff
@@ -964,7 +968,215 @@ def is_mod_loaded_for_ss(broker):
     yum_log = simple_file("/var/log/yum.log")
     yum_repolist = simple_command("/usr/bin/yum -C --noplugins repolist")
     yum_repos_d = glob_file("/etc/yum.repos.d/*.repo")
-    yum_updateinfo = simple_command("/usr/bin/yum -C updateinfo list")
+    #yum_updateinfo = simple_command("/usr/bin/yum -C updateinfo list")
+
+    @datasource(HostContext)
+    def yum_updates(_broker):
+        DEBUG = True
```

Comment:
DEBUG = True
should be better switched off for production

Reply:
debug  is still on

Reply:
Removed